docs/home.md

```diff
@@ -7,113 +7,66 @@
 
-Model repositories may reside on a locally accessible file system (e.g. NFS), as well as online storage compatible with 
-Google Cloud Storage (GCS), Amazon S3, or Azure Blob Storage. 
+## OpenVINO Model Server Key Features: 
 
-Read [release notes](https://github.com/openvinotoolkit/model_server/releases) to find out what’s new.
+The models used by OpenVINO Model Server need to be stored locally or hosted remotely by object storage services. For more details, refer to [Preparing Model Storage](./models_repository.md) documentation.  
 
-Review the [Architecture concept](architecture.md) document for more details.
+OpenVINO&trade; Model Server works with [Bare Metal Hosts](host.md) as well as [Docker containers](docker_container.md). It is also suitable for landing in the [Kubernetes environment](../deploy/README.md).
 
-Key features: 
-- support for multiple frameworks, such as Caffe, TensorFlow, MXNet, PaddlePaddle and ONNX
-- online deployment of new [model versions](model_version_policy.md)
-- [configuration updates in runtime](online_config_changes.md)
-- support for AI accelerators, such as 
+- Support for AI accelerators, such as 
 [Intel Movidius Myriad VPUs](https://docs.openvino.ai/2022.2/openvino_docs_OV_UG_supported_plugins_MYRIAD.html), 
 [GPU](https://docs.openvino.ai/2022.2/openvino_docs_OV_UG_supported_plugins_GPU.html), and 
 [HDDL](https://docs.openvino.ai/2022.2/openvino_docs_OV_UG_supported_plugins_HDDL.html) 
-- works with [Bare Metal Hosts](host.md) as well as [Docker containers](docker_container.md) 
-- [model reshaping](shape_batch_size_and_layout.md) in runtime
+- [model reshaping](shape_batch_size_and_layout.md) in runtime for high-throughput and low-latency
 - [directed Acyclic Graph Scheduler](dag_scheduler.md) - connecting multiple models to deploy complex processing solutions and reducing data transfer overhead
 - [custom nodes in DAG pipelines](custom_node_development.md) - allowing model inference and data transformations to be implemented with a custom node C/C++ dynamic library
-- [serving stateful models](stateful_models.md) - models that operate on sequences of data and maintain their state between inference requests
 - [binary format of the input data](binary_input.md) - data can be sent in JPEG or PNG formats to reduce traffic and offload the client applications
+- [serving stateful models](stateful_models.md) - models that operate on sequences of data and maintain their state between inference requests
+- online deployment of new [model versions](model_version_policy.md)
+- [configuration updates in runtime](online_config_changes.md) 
+- [performance tuning](performance_tuning.md)
 - [model caching](model_cache.md) - cache the models on first load and re-use models from cache on subsequent loads
 - [metrics](metrics.md) - metrics compatible with Prometheus standard
 
-**Note:** OVMS has been tested on RedHat, CentOS, and Ubuntu. The latest publicly released docker images are based on Ubuntu and UBI.
```

Comment:
can we remove this note or place it somewhere else? looks misplaced on the first page